src/util.js

```diff
@@ -203,20 +208,32 @@ function unpackToken(token) {
 function fullUrl(request) {
 	const protocol = request.protocol;
 	const host = request.host;
-	const path = request.originalUrl;
+	const opath = request.originalUrl;
 
-	return `${protocol}://${host}${path}`;
+	return `${protocol}://${host}${opath}`;
 }
 
 async function uploadCDNAsset(bucket, key, data, acl) {
-	const awsPutParams = {
-		Body: data,
-		Key: key,
-		Bucket: bucket,
-		ACL: acl
-	};
+	if (disabledFeatures.s3) {
+		await writeLocalCDNFile(key, data);
+	} else {
+		const awsPutParams = {
+			Body: data,
+			Key: key,
+			Bucket: bucket,
+			ACL: acl
+		};
+
+		await s3.putObject(awsPutParams).promise();
+	}
+}
+
+async function writeLocalCDNFile(key, data) {
+	const filePath = `${__dirname}/../cdn/${key}`;

```

Comment:
This should come from the config, so that I can docker-volume it out

Reply:
I'm assuming you mean the path where the CDN contents is written?

Reply:
Yeah, the filesystem path where they're written and read from